apps/serialization/messages.proto

```diff
@@ -281,6 +282,11 @@ enum TrapStatus{
   USED = 3;
 }
 
+enum PoolStatus{
+  ACTIVATING = 0;
+  ACTIVATED = 1;
```

Comment:
I think the following are better names, what do you think?
```suggestion
  WAITING = 0;
  READY = 1;
```

Reply:
completely agreed

Reply:
done!